server/app/views/applicant/ProgramIndexView.java

```diff
@@ -423,10 +423,10 @@ private LiTag programCard(
                 .withClasses(ReferenceClasses.APPLICATION_CARD_TITLE, "text-lg", "font-semibold")
                 .withText(program.localizedName().getOrDefault(preferredLocale));
     ImmutableList<DomContent> descriptionContent =
-        TextFormatter.createLinksAndEscapeText(
+        TextFormatter.formatText(
```

Comment:
This formats the program description on the home page for an applicant. 
<img width="1060" alt="image" src="https://github.com/civiform/civiform/assets/24319951/60372114-d4a4-48bd-b09d-b030a4467e4f">